app/src/test/java/org/fossasia/openevent/app/presenter/EventDetailPresenterTest.java

```diff
@@ -192,4 +191,18 @@ public void shouldNotAccessView() {
 
         verifyZeroInteractions(eventDetailView);
     }
+
+    @Test
+    public void shouldHideProgressbarCorrectly() {
+        when(eventRepository.getAttendees(id, false))
+            .thenReturn(Observable.fromIterable(attendees));
+
+        when(eventRepository.getEvent(id, false))
+            .thenReturn(Observable.just(event));
+
+        eventDetailPresenter.start();
+
+        assertTrue(eventDetailPresenter.getProgress() >= 2);
+        verify(eventDetailView).showProgressBar(false);
```

Comment:
Move the progress bar showing hiding to single test and remove the show code from above two tests